### PR TITLE
Build what-are-containers

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Builds what-are-containers as per [copydoc](https://docs.google.com/document/d/1hStuIzY8dDdFEW4Q0TwboOx0_-SB5klk8zRuNIn3V4o/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/containers/what-are-containers or https://ubuntu-com-10761.demos.haus/containers/what-are-containers
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against copydoc
- Check navigation works

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4567

### CODE REVIEW

File: templates/containers/what-are-containers.html [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+
+{% block content %}
+
+<section class="p-strip--suru-bottomed">
+    <div class="row u-equal-height">
+      <div class="col-7">
+        <h1>What are containers?</h1>
+        <p>Containers are a virtualisation technology that isolates each process in a lightweight and standard environment. They allow users to package and quickly deploy entire applications parallel to each other on the same kernel and hardware, while maintaining isolation among the workloads.</p>
+      </div>
+      <div class="col-5 u-align--center u-vertically-center">
```
Comment (minkyngkm) [inline templates/containers/what-are-containers.html:19]: I think we normally hide images on small screens.

```suggestion
      <div class="col-5 u-align--center u-vertically-center u-hide--small">
```

File: templates/containers/what-are-containers.html [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+    <p>Inside Google alone, at least <a href="https://cloud.google.com/containers/" class="p-link--external">two billion containers are generated each week</a> to manage its vast operations. There are many tools to create, deploy and manage containers, including LXD, Docker and Kubernetes.</p>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>What is Docker?</h2>
+    <p><a href="https://www.docker.com/" class="p-link--external">Docker</a> is an open-source containerisation platform. It provides an easy way to build and deploy containers on the cloud or on-premises. Container technologies were not introduced by Docker, but they made the user experience of container technologies so easy for developers that Docker soon became shorthand for containers as well as the most popular container format. <a href="https://www.docker.com/" class="p-link--external">Docker Inc</a>, the company that supports the Docker project and community, also operates <a href="https://hub.docker.com/" class="p-link--external">Docker Hub</a>, a space for hosting container images.</p>
```
Comment (minkyngkm) [inline templates/containers/what-are-containers.html:45]: Some words/sentences are different from copy doc. 

```suggestion
    <p><a href="https://www.docker.com/" class="p-link--external">Docker</a>  is an open-source containerisation platform. It provides an easy way to build and deploy containers in the cloud or on-premises. Docker didn’t introduce container technologies, but they made the user experience of container technologies so easy for developers that Docker soon became shorthand for containers and the most popular container format. <a href="https://www.docker.com/" class="p-link--external">Docker Inc</a>,the company that supports the Docker project and community, also operates <a href="https://hub.docker.com/" class="p-link--external">Docker Hub</a>, a space for hosting container images.</p>
```

File: templates/containers/what-are-containers.html [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+  </div>
+</section>
+
```
Comment (minkyngkm) [inline templates/containers/what-are-containers.html:48]: There is a missing section - "Docker for beginners: docker-cli, containerd, and runc"  here between "What is Docker?" and "Kubernetes vs. Docker: CRI-O and OCI
". "Docker for beginners: docker-cli, containerd, and runc"

File: templates/containers/what-are-containers.html [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+
+<section class="p-strip u-no-padding--top u-no-padding--bottom">
+  <div class="p-strip--suru-topped">
```
Comment (minkyngkm) [inline templates/containers/what-are-containers.html:79]: This looks a bit awkward to have `p-strip--suru-topped` for me. Would it be bette to remove this class? or do you have a design I can refer for this page by any chance? :)

File: templates/containers/what-are-containers.html [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-6">
+      <p>Not clear on how Kubernetes vs Docker differ?</p>
+      <p><a href="/blog/kubernetes-versus-docker">Read the Kubernetes vs Docker blog&nbsp;&rsaquo;</a></p>
+    </div>
+    <div class="col-6 u-align--center u-vertically-center">
```
Comment (minkyngkm) [inline templates/containers/what-are-containers.html:63]: I cannot find this image on the copy doc.

**Review by minkyngkm — CHANGES_REQUESTED** [review]
Hi Pete! Looking good mostly! I just left some comments for things I've found :)

File: templates/containers/what-are-containers.html (hunk 2 above)
Comment (petesfrench) [reply templates/containers/what-are-containers.html:45]: Good catch!

File: templates/containers/what-are-containers.html (hunk 4 above)
Comment (petesfrench) [reply templates/containers/what-are-containers.html:79]: You totally right, I am not sure why it is like this, but it is what is used in the [examples](https://ubuntu.com/0_docs/strips). Could be worth looking into more and filling an issue with vanilla.

File: templates/containers/what-are-containers.html (hunk 5 above)
Comment (petesfrench) [reply templates/containers/what-are-containers.html:63]: We made some changes in a call, I will add them to the doc :)

File: templates/containers/what-are-containers.html [hunk 6]
```diff
@@ -0,0 +1,250 @@
...
+
+{% block content %}
+
+<section class="p-strip--suru-bottomed">
+    <div class="row u-equal-height">
+      <div class="col-7">
+        <h1>What are containers?</h1>
+        <p>Containers are a virtualisation technology that isolates each process in a lightweight and standard environment. They allow users to package and quickly deploy entire applications parallel to each other on the same kernel and hardware, while maintaining isolation among the workloads.</p>
+      </div>
+      <div class="col-5 u-align--center u-vertically-center u-hide-small">
```
Comment (minkyngkm) [inline templates/containers/what-are-containers.html:19]: I think two dashes are needed here! :)
```suggestion
      <div class="col-5 u-align--center u-vertically-center u-hide--small">
```

File: templates/containers/what-are-containers.html [hunk 7]
```diff
@@ -0,0 +1,250 @@
...
+  </div>
+</section>
+
+<section class="p-strip ">
+  <div class="u-fixed-width">
+    <h2>Kubernetes vs Docker: CRI-O and OCI</h2>
+    <p>With more and more applications deployed on containers, simple orchestration tools are not enough to support complex production environments. ‘<a href="/kubernetes/what-is-kubernetes#container-orchestration">Container orchestration</a>’ becomes a problem space of its own. The rise of <a href="/kubernetes">Kubernetes</a> is the industry's answer to the container orchestration problem, providing a way to manage containers at scale efficiently.</p>
```
Comment (minkyngkm) [inline templates/containers/what-are-containers.html:59]:
```suggestion
    <p>With more and more applications deployed on containers, simple orchestration tools are not enough to support complex production environments. "<a href="/kubernetes/what-is-kubernetes#container-orchestration">Container orchestration</a>" becomes a problem space of its own. The rise of <a href="/kubernetes">Kubernetes</a> is the industry's answer to the container orchestration problem, providing a way to manage containers at scale efficiently.</p>
```

File: templates/containers/what-are-containers.html [hunk 8]
```diff
@@ -0,0 +1,250 @@
...
+      <p>Container images are made of software from many different projects and sources, not all equally well-maintained. Developers need to ask themselves what content can developers rely on today, tomorrow and for a long time thereafter. Particular attention should be paid to the base layer, specifically in selecting a frequently updated one with credible maintenance commitment, a large software ecosystem of well-maintained packages, and a great developer experience.</p>
+      <p>Canonical publishes a portfolio of base, runtime, and application images with an up to 10-year maintenance commitment. In collaboration with Docker, this content is publicly available on Docker Hub.</p>
+      <p><a href="/security/docker-images">Secure your software supply chain with LTS Docker Images&nbsp;&rsaquo;</a></p>
+    </div>
+    <div class="col-4 u-vertically-center u-align--center">
```
Comment (minkyngkm) [inline templates/containers/what-are-containers.html:147]:
```suggestion
    <div class="col-4 u-vertically-center u-align--center u-hide--small">
```

**Review by minkyngkm — APPROVED** [review]
LGTM! 👍

**Pull request merged**